### PR TITLE
cs-CZ: Apply #1584

### PR DESCRIPTION
**Pull request by octaroot**
cs-CZ: Apply #1584

### CODE REVIEW

File: data/language/cs-CZ.txt [hunk 1]
```diff
@@ -2225,7 +2225,7 @@ STR_2896    :{WINDOW_COLOUR_2}Hypothermia:   (Allister Brimble) copyright © Chr
 STR_2897    :{WINDOW_COLOUR_2}Last Sleigh Ride:   (Allister Brimble) copyright © Chris Sawyer
 STR_2898    :{WINDOW_COLOUR_2}Pipes of Glencairn:   (Allister Brimble) copyright © Chris Sawyer
 STR_2899    :{WINDOW_COLOUR_2}Traffic Jam:   (Allister Brimble) copyright © Chris Sawyer
-STR_2901    :{WINDOW_COLOUR_2}(Samples courtesy of Spectrasonics ”Liquid Grooves”)
```
[the remark this answers is not in the document]
Comment (octaroot) [reply data/language/cs-CZ.txt]: Done, I grep'd through the file and unified all quotation marks (such as ' or " or English ””)

File: data/language/cs-CZ.txt [hunk 2]
```diff
@@ -3606,7 +3606,7 @@ STR_6134    :Odstranit kulisy
 STR_6135    :Klient zaslal neplatný požadavek
 STR_6136    :Server zaslal neplatný požadavek
 STR_6137    :OpenRCT2 je zdarma dostupný open source klon hry Roller Coaster Tycoon 2.
-STR_6138    :OpenRCT2 je dílem mnoha lidí. Seznam všech přispěvatelů je dostupný v souboru “contributors.md”. Pro více informací navštivte http://github.com/OpenRCT2/OpenRCT2
+STR_6138    :OpenRCT2 je dílem mnoha lidí. Seznam všech přispěvatelů je dostupný v souboru „contributors.md”. Pro více informací navštivte http://github.com/OpenRCT2/OpenRCT2
```
Comment (Gymnasiast) [inline data/language/cs-CZ.txt]: „contributors.md__“__

File: data/language/cs-CZ.txt [hunk 3]
```diff
@@ -2324,7 +2324,7 @@ STR_3063    :{SMALLFONT}{BLACK}Vodní kanál (trať potopena)
 STR_3064    :Začátečnické parky
 STR_3065    :Středně obtížné parky
 STR_3066    :Parky pro experty
-STR_3067    :“Skutečné” parky
+STR_3067    :„Skutečné” parky
```
Comment (Gymnasiast) [inline data/language/cs-CZ.txt]: Closing mark is still English.

File: data/language/cs-CZ.txt [hunk 4]
```diff
@@ -2114,7 +2114,7 @@ STR_2781    :{STRINGID}:{MOVE_X}{195}{STRINGID}
 STR_2782    :SHIFT + 
 STR_2783    :CTRL + 
 STR_2784    :Změnit klávesovou zkratku
-STR_2785    :{WINDOW_COLOUR_2}Stiskněte novou klávesovou zkratku pro:{NEWLINE}“{STRINGID}”
+STR_2785    :{WINDOW_COLOUR_2}Stiskněte novou klávesovou zkratku pro:{NEWLINE}„{STRINGID}”
```
Comment (Gymnasiast) [inline data/language/cs-CZ.txt]: Closing mark is still English.

File: data/language/cs-CZ.txt [hunk 5]
```diff
@@ -1462,9 +1462,9 @@ STR_2086    :Kupon na {STRINGID}
 STR_2087    :Popcorn
 STR_2088    :Párek v rohlíku
 STR_2089    :Chapadlo
-STR_2090    :“{STRINGID}” čepice
+STR_2090    :„{STRINGID}” čepice
 STR_2091    :Jablko v županu
-STR_2092    :“{STRINGID}” tričko
+STR_2092    :„{STRINGID}” tričko
```
Comment (Gymnasiast) [inline data/language/cs-CZ.txt]: Closing mark is still English.

File: data/language/cs-CZ.txt [hunk 6]
```diff
@@ -1462,9 +1462,9 @@ STR_2086    :Kupon na {STRINGID}
 STR_2087    :Popcorn
 STR_2088    :Párek v rohlíku
 STR_2089    :Chapadlo
-STR_2090    :“{STRINGID}” čepice
+STR_2090    :„{STRINGID}” čepice
```
Comment (Gymnasiast) [inline data/language/cs-CZ.txt]: Closing mark is still English.

File: data/language/cs-CZ.txt [hunk 7]
```diff
@@ -1444,11 +1444,11 @@ STR_2068    :smažené kuře
 STR_2069    :limonáda
 STR_2070    :prázdná krabička
 STR_2071    :prázdná lahev
-STR_2072    :“{STRINGID}” balónek
-STR_2073    :“{STRINGID}” plyšák
+STR_2072    :„{STRINGID}” balónek
```
Comment (Gymnasiast) [inline data/language/cs-CZ.txt]: Closing mark is still English. (x3)

File: data/language/cs-CZ.txt [hunk 8]
```diff
@@ -461,14 +461,14 @@ STR_1076    :Mód stánku/obchodu
 STR_1077    :Mód rotace
 STR_1078    :Dopředná rotace
 STR_1079    :Zpětná rotace
-STR_1080    :Film: ”Pomsta letců”
-STR_1081    :3D film: ”Myší ocásky”
+STR_1080    :Film: „Pomsta letců”
```
Comment (Gymnasiast) [inline data/language/cs-CZ.txt]: Closing mark is still English. (x5)

**Comment by octaroot** [comment]
Unified visually similar quotation marks, I'll now squash the commits.

File: data/language/cs-CZ.txt [hunk 9]
```diff
@@ -2080,11 +2080,11 @@ STR_2739    :Žádná
 STR_2740    :RollerCoaster Tycoon 1
 STR_2741    :RollerCoaster Tycoon 2
 STR_2742    :css50.dat nenalezen
-STR_2743    :Nakopírujte 'data/css17.dat' z vaší instalace RCT1 do 'data/css50.dat' ve vaší instalaci RCT2.
+STR_2743    :Nakopírujte „data/css17.dat“ z vaší instalace RCT1 do „data/css50.dat“ ve vaší instalaci RCT2.
 STR_2744    :[
 STR_2745    :\
 STR_2746    :]
-STR_2747    :”
+STR_2747    :“
```
Comment (Gymnasiast) [inline data/language/cs-CZ.txt]: Should not have been replaced.

Comment (octaroot) [reply data/language/cs-CZ.txt]: fixed

**Pull request merged**